website/docs/patterns/button-alignment/partials/overview/overview.md

```diff
@@ -0,0 +1,98 @@
+While the alignment, order, and grouping of buttons and actions might seem trivial, the impact on users is immediate and often occurs in experiences of high important and interruption.
+
+These guidelines don't cover every scenario and are intended to aid in the decision-making process when determining how to organize buttons and actions. Different methods of alignment and ordering will result in different outcomes with pros and cons for each. What is most important is **consistency** across an application in how actions are aligned, ordered and grouped.
+
+!!! Info
+
+Button alignment isn't relevant for every component or pattern, but is rather aimed at functional elements; i.e. modals, forms, and page-level actions.
+!!!
+
+## Constrained vs. unconstrained
+
+**Constrained** refers to a component or pattern that is wrapped within a container element, while **unconstrained** refers to components that exist at the page level.
+
+- **Constrained**
+    - Dialogs and modals
+    - Toolbars
+    - Alerts
+    - Banners
+- **Unconstrained**
+    - General page content
+    - In-page forms
+    - The "window" in a general sense
+
+This differentiation is important because the expectations from the user are different in each context. A constrained component or pattern will often be used in an interstitial or circumstantial experience; i.e. an experience that is triggered by the user or the application illiciting a quick, progressive action.
+
+Unconstrained generally refers to a more "static" experience that isn't triggered by a user action. This is often tied to a page-level function or a navigational element.
+
+**Insert some supporting images here**
```

Comment:
For a note like this, you could put a TODO comment instead so that it wouldn't be visible to people visiting the site in case it gets committed.